tekton/resources/nightly-release/base/template.yaml

```diff
@@ -17,21 +17,4 @@ spec:
     default: gcr.io/tekton-nightly
   - name: projectName
     description: Name of the Tekton project to release (e.g. pipeline, triggers, etc).
-  resourcetemplates:
-  - apiVersion: tekton.dev/v1alpha1
-    kind: PipelineResource
-    metadata:
-      name: tekton-bucket-nightly-$(tt.params.projectName)
-    spec:
-      params:
-      - name: type
-        value: gcs
-      - name: location
-        value: gs://tekton-releases-nightly/$(tt.params.projectName)
-      - name: dir
-        value: "y"
-      type: storage
-      secrets:
-      - fieldName: GOOGLE_APPLICATION_CREDENTIALS
-        secretKey: service-account.json
-        secretName: nightly-account
+  resourcetemplates:
```

Comment:
haven't used kustomize before...does this look ok @afrittoli ? 
Do we need to keep `resourcetemplates` with no values here?

Reply:
```
Error: accumulating resources: recursed accumulation of path '/System/Volumes/Data/go/src/github.com/tektoncd/plumbing/tekton/resources/nightly-release': accumulating resources: recursed accumulation of path '/System/Volumes/Data/go/src/github.com/tektoncd/plumbing/tekton/resources/nightly-release/overlays/dashboard': failed to apply json patch '[{"op":"add","path":"/spec/resourcetemplates/-","value":{"apiVersion":"tekton.dev/v1beta1","kind":"PipelineRun","metadata":{"name":"dashboard-release-nightly-$(uid)"},"spec":{"params":[{"name":"package","value":"$(tt.params.gitrepository)"},{"name":"imageRegistry","value":"gcr.io/tekton-nightly"},{"name":"versionTag","value":"$(tt.params.versionTag)"},{"name":"bucketName","value":"latest"}],"pipelineRef":{"name":"dashboard-release"},"resources":[{"name":"dashboard-source-repo","resourceSpec":{"params":[{"name":"revision","value":"$(tt.params.gitrevision)"}],"type":"git"}},{"name":"bucket-for-dashboard","resourceSpec":{"params":[{"name":"type","value":"gcs"},{"name":"location","value":"gs://tekton-releases-nightly/$(tt.params.projectName)"},{"name":"dir","value":"y"}],"secrets":[{"fieldName":"GOOGLE_APPLICATION_CREDENTIALS","secretKey":"service-account.json","secretName":"nightly-account"}],"type":"storage"}},{"name":"builtDashboardImage","resourceRef":{"name":"dashboard-image"}}]}}}]': add operation does not apply: doc is missing path: "/spec/resourcetemplates/-": missing value
```

I tried both removing the `-` from the path in the `add` operation as well as adding one to the template but none of them seem to work...

Reply:
Hmmm... 🤔

Reply:
Ok, hacky fix but instead of appending to a non-existent list, I'm now just adding the entire resourceTemplates section in the overlay.

Reply:
At least, on my local env, this seems to fix the issue